### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jacoffey3**
Fleshed out the readme to help new people (and Trial users)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,67 @@
-# example-python-gateway
-Example implementation of a Python Gateway for Major Tom. Clone locally to use it with Major Tom.
+# Demo Python Gateway
+
+This repository serves as both an example implementation of how a Gateway interacts with Major Tom using the Gateway API and
```
Comment (marshall) [inline README.md]:
```suggestion
This Gateway serves as both an example of interacting with Major Tom using the Gateway API and 
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,67 @@
-# example-python-gateway
-Example implementation of a Python Gateway for Major Tom. Clone locally to use it with Major Tom.
+# Demo Python Gateway
+
+This repository serves as both an example implementation of how a Gateway interacts with Major Tom using the Gateway API and 
+a demo Satellite which responds to commands, generates telemetry, and uplinks/downlinks files. 
+
+If you have any issues setting up/running this demo, 
+either [make and issue](https://github.com/kubos/example-python-gateway/issues/new) on this repository
```
Comment (marshall) [inline README.md]:
```suggestion
either [make an issue](https://github.com/kubos/example-python-gateway/issues/new) on this repository
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,67 @@
...
 ```pip3 install -r requirements.txt```
 
-View how to use by accessing the help:
+## Major Tom Setup
+
+In addition to setting up your Local Environment, you'll need to make a Gateway in Major Tom! 
+Once you've received your login credentials for Major Tom, make a new mission, 
+and you'll be prompted to add a Gateway to that mission. 
+
+Once you add the Gateway, make a note of the Gateway's __Authentication Token__ listed on the Gateway Page.
```
Comment (marshall) [inline README.md]: can you add a screenshot that has a circle around the page where this comes from?

File: README.md [hunk 4]
```diff
@@ -1,10 +1,67 @@
...
 
 ```python3 run.py -h```
+
+If you are using this on a __Trial__ of Major Tom, you will most likely run this command: 
+
+```python3 run.py -m app.majortom.cloud -g {YOUR-GATEWAY-AUTHENTICATION-TOKEN}```
+
+For Example: 
+
+```python3 run.py -m app.majortom.cloud -g 4b1273101901225a9d3df4882884b26e139cdeb49d2c1a50a51baf66c3a42623```
```
Comment (marshall) [inline README.md]: is this a valid token?

File: README.md [hunk 5]
```diff
@@ -1,10 +1,67 @@
...
+
+Once you run this, should should see Major Tom respond with a `hello` message:
+
+```2019-08-19 12:04:46,151 - major_tom.major_tom - INFO - Major Tom says hello: {'type': 'hello', 'hello': {'mission': 'Demo'}}```
```
Comment (marshall) [inline README.md]: does this log message appear in the Gateway console, or in the Major Tom console?

File: README.md [hunk 6]
```diff
@@ -1,10 +1,67 @@
...
+
+## What does this Demo Satellite do? 
+
+Now that you've connected the gateway, it will automatically create a new satellite named "Space Oddity" and load in command definitions for it. 
+
+You can now issue those commands to the satellite through the connected Gateway, which accomplish a wide variety of simulated tasks, all of which are explained by the description under each command. 
+
+To find these commands, go to Major Tom under the mission you originally created and hover over the satellites icon on the sidebar. 
+Space Oddity should be in the slider menu, and clicking on it will take you to its commanding page.
```
Comment (marshall) [inline README.md]: another good place to insert screenshot(s)

File: README.md (hunk 4 above)
Comment (jacoffey3) [reply README.md]: No

File: README.md (hunk 5 above)
Comment (jacoffey3) [reply README.md]: Gateway

**Comment by jacoffey3** [comment]
[Onboarding docs](https://trello.com/c/bAfq5pqq/1697-onboarding-docs)

**Pull request merged**